README.md

```diff
@@ -165,6 +165,7 @@ Additional lists you might find useful:
 *Plugins for file manipulation.*
 
 - [FileStorage plugin](https://github.com/burzum/cakephp-file-storage) - Abstract file storage and upload plugin.
+- [Filesystem plugin](https://github.com/josbeir/cakephp-filesystem) - Abstract [Flysystem](http://flysystem.thephpleague.com/) + file entity based upload plugin.
```

Comment:
Isnt the name technically `Josbeir/Filesystem` ?

Reply:
Technically yes but its a bit weird in representation? Are you proposing something? ;-)

Reply:
Well, that's the name and thus the standard display name we are using to not confuse people.

Reply:
Fixed !